test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGetPackageManagerTests/NuGetPackageManagerTelemetryTests.cs

```diff
@@ -147,15 +147,14 @@ await nuGetPackageManager.PreviewInstallPackageAsync(
                     CancellationToken.None);
 
                 // Assert
-                Assert.Equal(3, telemetryEvents.Count);
                 Assert.Equal(2, telemetryEvents.Count(p => p.Name == "ProjectRestoreInformation"));
                 Assert.Equal(1, telemetryEvents.Count(p => p.Name == ActionTelemetryStepEvent.NugetActionStepsEventName));
 
                 Assert.Contains(telemetryEvents.Where(p => p.Name == ActionTelemetryStepEvent.NugetActionStepsEventName), p => (string)p["SubStepName"] == TelemetryConstants.PreviewBuildIntegratedStepName);
 
                 var projectFilePaths = telemetryEvents.Where(p => p.Name == "ProjectRestoreInformation").SelectMany(x => x.GetPiiData()).Where(x => x.Key == "ProjectFilePath");
                 Assert.Equal(2, projectFilePaths.Count());
-                Assert.True(projectFilePaths.All(p => p.Value is string y && File.Exists(y) && (y.EndsWith(".csproj") || y.EndsWith("project.json") || y.EndsWith("proj"))));
+                Assert.True(projectFilePaths.All(p => p.Value is string y && (y.EndsWith(".csproj") || y.EndsWith("project.json") || y.EndsWith("proj"))));
```

Comment:
no point in checking the file exists. The test is not operating on it.

Reply:
Should we also remove the "project.json" path check?

Reply:
done.